languages/python/sqlalchemy-oso/tests/test_sqlalchemy.py

```diff
@@ -273,11 +274,14 @@ def test_null_with_partial(engine, oso):
     )
     posts = Session().query(Post)
 
-    assert str(posts) == (
-        "SELECT posts.id AS posts_id, posts.contents AS posts_contents, posts.title AS posts_title, "
-        + "posts.access_level AS posts_access_level, posts.created_by_id AS posts_created_by_id, "
-        + "posts.needs_moderation AS posts_needs_moderation \nFROM posts \nWHERE posts.contents IS NULL"
-    )
+    # TODO(gj): 'do_orm_execute' event only fires when a query is executed; not
```

Comment:
FYI @dhatch. I'll remove this block once you see it — just thought it was a worthwhile thing to note that's different from the `before_compile` event.

Reply:
Why is this only an issue here, but not on the other string checks throughout the tests?

Reply:
38f84b88

Reply:
Sounds good, leaving this comment would be helpful.

Reply:
What do you mean? I left comments inline in the code (see the linked patch).